contracts/AssetPool.sol

```diff
@@ -391,6 +391,21 @@ contract AssetPool is Ownable, IAssetPool {
         withdrawalTimeoutChangeInitiated = 0;
     }
 
+    /// @notice Grants pool shares by minting a given amount of the underwriter
+    ///         tokens for the recipient address. In result, the recipient
+    ///         obtains part of the pool ownership without depositing any
+    ///         collateral tokens.
```

Comment:
Worth adding a note in what circumstances this function could be used (rewards for various contract state notifiers).

Reply:
See https://github.com/keep-network/coverage-pools/pull/111/commits/5b3b64029f99da1394f103cc2c7db4f05cdc7966